hardware/edgeconnector_ds.md

```diff
@@ -218,10 +218,10 @@ directly to the pin. These devices often have significant back-EMF
 when energised, and will generate voltages that exceed the maximum
 specifications of the GPIO pins, and may cause premature device failure.
 
-NOTE 3: The pin marked 'reserved for accessibility' is not presently
-used, and may be used as a general purpose input/output pin.
-It may be used in future releases of the software runtime, to enable
-and disable an on-board accessibility mode.
+NOTE 3: The pin marked 'ACCESSIBILITY' is used to enable/disable an on-board 
+accessibility mode, and should not be used for anything else (even though it
+can be used as a GPIO for testing). Future versions of the official micro:bit
+editors may remove the ability to write to this pin.
```

Comment:
Even if we are repeating ourselves, I would probably add this note as well to the "Pins and Signals" Notes section, which starts on line 95 ([dev-docs/hardware/edgeconnector_ds.md#L95](https://github.com/microbit-foundation/dev-docs/blame/cb53ab6b892f74a4739d573ac0c9591dcfb9c000/hardware/edgeconnector_ds.md#L95)).